lib/components/input-group.vue

```diff
@@ -1,19 +1,25 @@
 <template>
-    <component :is="tag" :class="classObject" role="group">
+    <component :is="tag" :id="id || null" :class="classObject" role="group">
```

Comment:
In this case ID is not required.  Would the validator crap out (invalid when passed null or am empty string?

Vue typically will add the attribute if passed an empty string.  The `|| null` ensures that the attribute will not appear if anything falsey is given (`null`, `''`. `0`, etc)